.github/workflows/ruff.yml

```diff
@@ -0,0 +1,14 @@
+# https://beta.ruff.rs
+name: ruff
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: pip install --user ruff
+    - run: ruff --format=github .
```

Comment:
The GitHub code annotations will conduct by adding the `--format=github` option? I did not know about this feature yet, thank you so much! Ruff is an excellent piece of software, we love it.

/cc @charliermarsh

Reply:
Yes.  https://beta.ruff.rs/docs/settings/#format

Reply:
💡  You can also use the `RUFF_OUTPUT_FORMAT` environment variable, which is useful if `ruff` is embedded in e.g. a `pre-commit` flow:

```yaml
run: pre-commit run --all-files
env:
  RUFF_OUTPUT_FORMAT: github
```

Reply:
It was renamed in September (https://github.com/astral-sh/ruff/pull/7514) and the old name deprecated in October (https://github.com/astral-sh/ruff/pull/7984). 

This PR has been pending since April... 😁